README.md

```diff
@@ -12,13 +12,13 @@ The contents of this code of conduct apply to our interactions in various areas
 
 ## Values
 
-We want our team to be a fun, productive, and safe space for all members. In addition to core values of respect and honesty, there are several ways in which our product team distinguishes itself.
+We want our team to be fun, productive, and safe for all members. In addition to the Social Tables [core values](https://www.socialtables.com/company/about/#values), there are several ways in which our Engineering team distinguishes itself.
```

Comment:
Does this CoC apply to Engineering only, or should we bring in Product and/or Design as well? Engineering interfaces almost exclusively with them, and rather tightly as well.